robot/client/client.go

```diff
@@ -933,8 +933,8 @@ func (rc *RobotClient) Log(ctx context.Context, log zapcore.Entry, fields []zap.
 	return err
 }
 
-// GetCloudMetadata returns app-related information about the robot.
-func (rc *RobotClient) GetCloudMetadata(ctx context.Context) (cloud.Metadata, error) {
+// CloudMetadata returns app-related information about the robot.
+func (rc *RobotClient) CloudMetadata(ctx context.Context) (cloud.Metadata, error) {
```

Comment:
Is this rename breaking in any sense?

Reply:
Also curious -- is there a consequence to not doing the rename? It's unclear if the motivation is technical or consistency/style.

Reply:
consistency/style per @cheukt

Reply:
I don't feel too strongly, and I'm happy to do it here since you notified the breaking changes channel 👍🏻 .

Reply:
Is this one of those "fake" public APIs? Where we use it internally, but it's not really for users writing robot code?

Reply:
> Is this one of those "fake" public APIs? Where we use it internally, but it's not really for users writing robot code?

@cheukt do you recall the use-cases where having this API on the robot is convenient?

Reply:
it'll be useful for querying data from app, makes it easier to tie the robot you're scripting against to data from the robot/location/org